### PR TITLE
Add Feature1 

### PR DESCRIPTION
**Pull request by echo-ech0**
Feature1 description

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 console.log("Index file");
 console.log("Index 2 added");
 console.log("Index 3 added");
+console.log("Index 4 added");
```
Comment (SubhajitBar) [inline index.js:4]: message 1

Comment (SubhajitBar) [inline index.js]: review1

File: index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 console.log("Index file");
 console.log("Index 2 added");
 console.log("Index 3 added");
```
Comment (SubhajitBar) [inline index.js:3]: review 2

File: index.js (hunk 1 above)
Comment (echo-ech0) [reply index.js]: code has been changed. please review

**Review by SubhajitBar — APPROVED** [review]

**Pull request merged**